src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_StringInterpolation.cs

```diff
@@ -220,8 +220,9 @@ private void MakeInterpolatedStringFormat(BoundInterpolatedString node, out Boun
                 if (fillin == null)
                 {
                     Debug.Assert(part is BoundLiteral && part.ConstantValue != null);
-                    // this is one of the literal parts
-                    stringBuilder.Append(part.ConstantValue.StringValue);
+                    // this is one of the literal parts.  If it contains a { or } then we need to escape those so that
+                    // they're treated the same way in string.Format.
+                    stringBuilder.Append(ConstantValueUtils.EscapeInterpolatedStringLiteral(part.ConstantValue.StringValue));
```

Comment:
note: the EscapeInterpolatedStringLiteralis only called here. i could move that helper here instead :)

Reply:
In my cursory search, I could only find one test affected by this (`InterpolatedStringWithCurlyBracesFollowerAfterFormatSpecifierTest`). Do we have others?
Feels like we should cover this more. For example, what if we have a backslash in the literal part before the curly? What if we have double curlies in the string literal?